benchmark_runner/main/main.py

```diff
@@ -35,18 +35,22 @@ def main():
     install_ocp_version = environment_variables_dict.get('install_ocp_version', '')
     install_ocp_resources = environment_variables_dict.get('install_ocp_resources', '')
     run_type = environment_variables_dict.get('run_type', '')
+
+    is_benchmark_operator_workload = 'benchmark-operator' in (environment_variables.get_workload_namespace(workload), environment_variables_dict.get("runner_type"))
+    is_benchmark_runner_workload = 'benchmark-runner' in (environment_variables.get_workload_namespace(workload), environment_variables_dict.get("runner_type"))
     # workload name validation
     if workload and not ci_status:
         if workload not in environment_variables.workloads_list:
-            logger.info(f'Enter valid workload name {environment_variables.workloads_list}')
-            raise Exception(f'Not valid workload name: {workload} \n, choose one from the list: {environment_variables.workloads_list}')
+            #logger.info(f'Enter valid workload name {environment_variables.workloads_list}')
+            #raise Exception(f'Not valid workload name: {workload} \n, choose one from the list: {environment_variables.workloads_list}')
+            pass # ignore this error
```

Comment:
Why should this be ignored?

Reply:
hum, that's something unfinished, but my overall idea is to avoid having anything related to (new) workloads outside of the workload directory.

see this commit to see how I add my workload: https://github.com/kpouget/benchmark-runner/commit/0db6a70e2e3401112a90744d20ff6db256b4a937

I think this `workloads_list` is just a safe guard, maybe avoiding running "nothing" if the workload variable isn't properly configured

I could add another env var `ignore_workloads_list` to preserve the existing behavior

Reply:
I think for now we should not ignore the error, but please open an feature request about avoiding anything related to new workloads outside of the workload directory.  I agree with your reasoning.